projects/ppwcode/ng-common-components/src/lib/dashboard-items/dashboard-items-table/dashboard-items-table.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="dashboard-wrapper-container">
+    <div class="dashboard-wrapper">
+        @for (item of dashboardItems(); track item) {
+            <mat-card class="dashboard-item">
+                <mat-card-header
+                    [matBadge]="item.badgeContent"
+                    [matBadgeHidden]="!item.badgeContent?.length"
+                    [matBadgeOverlap]="true"
+                    matBadgePosition="above after"
+                    matBadgeSize="large"
+                >
+                    <mat-card-title>{{ item.cardTitleKey | translate }}</mat-card-title>
+                </mat-card-header>
+                @if (item.template) {
+                    <a
+                        href=""
+                        (click)="
+                            $event.stopPropagation(); $event.preventDefault(); executeAction.emit(item.defaultAction)
+                        "
+                        class="ppw-dashboard-anchor"
+                    >
+                        <ng-container [ngTemplateOutlet]="item.template"></ng-container>
+                    </a>
+                } @else {
+                    <i
+                        class="{{ item.iconClass }} ppw-dashboard-card-image"
+                        mat-ripple
+                        [tabindex]="0"
+                        (keyup.enter)="executeAction.emit(item.defaultAction)"
+                        (click)="executeAction.emit(item.defaultAction)"
+                    ></i>
+                }
+                <mat-card-content class="flex-grow-1">
```

Comment:
Is this class available by default? I would replace this with a class `ppw-dashboard-item-content` and do it in the scss of this component to avoid the dependency on the class.

Reply:
Same applies to the gap-small class added to the actions